Update Data Exploration (#26)
Caitlin1306/nch-final-project: README.md

## Data Exploration
 
[Link to my Deepnote](https://deepnote.com/workspace/caitlin1306-5830-b744c156-1215-41b0-afc0-861533a29a82/project/NCH-Bootcamp-f2ebe341-884e-42c0-b194-dc0e178c25ee/notebook/NCH%20Final%20Project%20-8e630e6780274171848330c518ff31c7)

> Data exploration, helps us to achieve a basic understanding of the chosen data. The results of data exploration can be extremely useful in understanding the structure of the dataset. Descriptive statistics is the process of condensing key characteristics of the data set into simple numeric metrics such as; mean, max and standard deviation etc.

Reviewer: ```suggestion
Data exploration helps us to achieve a basic understanding of the chosen data. The results of data exploration can be extremely useful in understanding the dataset's structure. Descriptive statistics condenses critical characteristics of the data set into simple numeric metrics such as mean, maximum, standard deviation, etc.
```
